package.json

```diff
@@ -30,11 +30,11 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@aws-sdk/client-route-53": "^3.261.0",
```

Comment:
Please avoid making cosmetic changes to files.

Reply:
I disagree, npm alphabetizes these, so we should follow the convention. Plus, it gets auto moved in certain cases.